doc/news/changes.h

```diff
@@ -223,6 +223,28 @@ inconvenience this causes.
 
 <ol>
 
+ <li> New: Implemented within the Physics::Elasticity namespace is a class
```

Comment:
> Implemented within.. is a class ... that...

something is wrong here, i guess you can just say `Implement several standard tensors ... within Physics::Elasticity::StandardTensors and Physics::Elasticity::Kinematics` or alike.